src/python/circllhist/__init__.py

```diff
@@ -0,0 +1,77 @@
+import circllhist.ffi as ffi
+
+
+class circllbin:
+    def __init__(self, b):
+        self.b = b
+
+    @classmethod
+    def new(cls):
+        return cls(ffi.ffi.new("hist_bucket_t*"))
+
+    @classmethod
+    def from_number(cls, n):
+        return cls(ffi.C.double_to_hist_bucket(n))
+
+    def __str__(self):
+        buf = ffi.ffi.new("char[]",9)
+        ffi.C.hist_bucket_to_string(self.b, buf)
+        return str(ffi.ffi.string(buf), "ASCII")
```

Comment:
`"ASCII"` encoding argument is Python 3 only. Should keep it compatible with both Python 2 and 3.

Something like this:
```diff
diff --git a/src/python/circllhist/__init__.py b/src/python/circllhist/__init__.py
index fc6abbe..9b2adab 100644
--- a/src/python/circllhist/__init__.py
+++ b/src/python/circllhist/__init__.py
@@ -1,5 +1,10 @@
 import circllhist.ffi as ffi
 
+import sys
+if sys.version_info[0] == 3:
+    str_ascii = lambda b: str(b, "ASCII")
+else:
+    str_ascii = str
 
 class circllbin:
     def __init__(self, b):
@@ -16,7 +21,7 @@ class circllbin:
     def __str__(self):
         buf = ffi.ffi.new("char[]",9)
         ffi.C.hist_bucket_to_string(self.b, buf)
-        return str(ffi.ffi.string(buf), "ASCII")
+        return str_ascii(ffi.ffi.string(buf))
 
```